server/src/main/java/com/cloud/network/router/CommandSetupHelper.java

```diff
@@ -1101,4 +1105,14 @@ protected String getGuestDhcpRange(final NicProfile guestNic, final Network gues
         }
         return dhcpRange;
     }
+
+    private TrafficType getNetworkTrafficType(Network network) {
+        final VpcGatewayVO gateway = _vpcGatewayDao.getVpcGatewayByNetworkId(network.getId());
+        if (gateway != null) {
+            s_logger.debug("network " + network.getId() + " is a vpc private gateway, set traffic type to Public");
```

Comment:
@ustcweizhou what do you think of adding also the network name? Sometimes it annoys me to see a lot of IDs on the log and then go after a few selects to understand what is really going on.

Having the network ID and name would be great for debugging :-)

Reply:
@GabrielBrascher added a new commit per your comment.